adafruit_pyportal.py

```diff
@@ -57,6 +57,7 @@
 import adafruit_requests as requests
 import storage
 import displayio
+from adafruit_display_text.Label import Label
```

Comment:
```suggestion
from adafruit_display_text.label import Label
```

Reply:
Yes, I think so. I'm not sure why it was able to work previously actually. 

New commit changes to lower case.